Civi/Api4/GroupContact.php

```diff
@@ -24,8 +24,9 @@
  * A contact can either be "Added" "Removed" or "Pending" in a group.
  * CiviCRM only considers them to be "in" a group if their status is "Added".
  *
+ * @bridge group_id contact_id
  * @see \Civi\Api4\Group
- *
+ * @searchable false
```

Comment:
I hadn't noticed this before -- so does this mean that metadata can be declared in different places (`xml/schema` vs class-annotations) -- and it'll get merged together for purposes `getFields` (etc)? That's pretty neat.

Reply:
Yes any annotation on an APIv4 entity class will get passed through to `Api4\Entity::get()`